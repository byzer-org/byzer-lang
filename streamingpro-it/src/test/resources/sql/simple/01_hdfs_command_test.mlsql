--%comparator=tech.mlsql.it.IgnoreResultComparator
!hdfs -ls /tmp;