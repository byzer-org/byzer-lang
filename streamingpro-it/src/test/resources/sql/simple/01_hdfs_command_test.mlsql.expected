fileSystem
Found 1 items[\\s\\S]+